configs/setups/awiesm/awiesm.yaml

```diff
@@ -128,24 +300,83 @@ fesom:
                 T63_REF:
                         nproc: 128
 
-        opbnd_dir: ""
-        tide_forcing_dir: ""
-        forcing_data_dir: ""
-        model_dir: ${general.model_dir}/fesom-${fesom.version}
-        setup_dir: ${general.model_dir}
-
-        add_namelist_changes:
-                namelist.oce:
-                        boundary:
-                                restore_s_surf: 0.0
-
-        leapyear: True
-        asforcing: ECHAM5
-
-
+        # with_pico switch
+        # ----------------
+        choose_with_pico:
+          True:
+            add_forcing_files:
+              pico_forcing: temp_salt_default
+            add_forcing_sources:
+              # FIXME(PG): Needs to be in a pool somewhere...
+              temp_salt_default: ${general.project_base}/nonstandard_forcings/pico/test.dat
+            add_namelist_changes:
+              namelist.forcing:
+                pico:
+                  use_pico: True
```

Comment:
the defaults are in both cases sensible in the hard-coded fortran, so I would not put an extra layer of defaults coming from the ESM-Tools. Too many layers of confusion otherwise.